include/pybind11/typing.h

```diff
@@ -0,0 +1,97 @@
+/*
+    pybind11/typing.h: Convenience wrapper classes for basic Python types
+    with more explicit annotations.
+
+    Copyright (c) 2016 Wenzel Jakob <wenzel.jakob@epfl.ch>
```

Comment:
Sorry I forgot to comment on this before, could you please either put your name & current year, or maybe:

```
// Copyright (c) 2023 The pybind Community.
```

(That's what I usually use.)